.github/workflows/other-image-test.yml

```diff
@@ -0,0 +1,121 @@
+name: Other Image Test
```

Comment:
Since there's a lot of copy-paste in here, I suppose there is no nice/easy way of combining it with our main python workflow?